backend/src/packages/auth/auth.service.ts

```diff
@@ -12,10 +15,20 @@ class AuthService {
     this.userService = userService;
   }
 
-  public signUp(
+  public async signUp(
     userRequestDto: UserSignUpRequestDto,
-  ): Promise<UserSignUpResponseDto> {
-    return this.userService.create(userRequestDto);
+  ): Promise<UserSignUpResponseDto | undefined> {
```

Comment:
```suggestion
  ): Promise<UserSignUpResponseDto> {
```
I think we can't have ```undefined``` here, since ```this.userService.create(userRequestDto);``` returns ```Promise<UserSignUpResponseDto>```